### ISSUE TITLE
Pouvoir présenter plusieurs contact en pied de page

### ISSUE THREAD

**Issue by dtrucs** [GeotrekCE/Geotrek-rando-v3#579]
Ajourd'hui, le bloc contact est un peu rigide car il permet de ne mettre qu'un contact avec une seule adresse, un email et un tel.

Cf la config en exemple (https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/frontend/customization/config/footer.json#L25)
```
  "contact": {
    "name": "Parc National des Écrins",
    "addressLine1": "Domaine de Charance",
    "addressLine2": "05000 Gap",
    "number": "04 92 40 20 10",
    "mail": "info@ecrins-parcnational.fr"
  }
```

Il serait intéressant de rendre cela plus souple, en transformant la valeur de la clé `contact` par un tableau d'objets et boucler dessus :

```
  "contact": [{
      "name": "Parc National des Écrins",
      "addressLine1": "Domaine de Charance",
      "addressLine2": "05000 Gap",
      "number": "04 92 40 20 10",
      "mail": "info@ecrins-parcnational.fr"
    },
    { ... }
  ]
```

Et pour aller plus loin: faire de même avec les les clés `number` et  `mail` de chaque contact  (c-à-d les convertir en tableau) .

À refléchir de la pertinence, p-e qu'un champ HTML libre garantit + de souplesse (cf https://github.com/GeotrekCE/Geotrek-rando-v3/issues/549)

**Comment by camillemonchicourt** [comment]
Oui le rendre plus souple serait intéressant.

**Comment by camillemonchicourt** [comment]
Hum en effet c'est un peu différent.
Et selon la doc (et les tests que j'avais fait de mémoire), on peut bien créer tous les liens que l'on veut en ajoutant les clés de langue dans ta surcouche de traduction, sans nécessité d'utiliser seulement ceux donnés en exemple : 

> - In the `footer.json` file, you can define social networks, informations about your organization, and some links (see example in https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/frontend/customization/config/footer.json).
> 
>   - Social networks: `facebook`, `twitter`, `youtube`, `instagram` or `fallback`.
>   - Contact information such as your name, address, phone number and email.
>   - Links based on the key pair `label`/`url` (can be based on translation labels for multilingual) and/or the key `informationID` whose value is equal to a flatpage identifier.

**Comment by mviadere-openig** [comment]
D'accord, merci pour cette information complémentaire.

**Comment by mviadere-openig** [comment]
Bonjour, je relance ce sujet car on m'a demandé récemment s'il était possible de mettre l'adresse de plusieurs offices de tourisme dans le footer. Est-ce que cette fonctionnalité a été implémenté depuis ?

**Comment by camillemonchicourt** [comment]
Non, pas de nouveauté, sinon on l'aurait indiqué ici, comme on fait habituellement sur les différents tickets pour que chacun puisse suivre.
Toute contribution est possible sur le sujet si vous souhaitez le faire avancer.
Sinon, tu peux toujours ne pas mettre de contact dans le FOOTER, vois ne pas utiliser le FOOTER fourni par défaut, et en faire un à la main en HTML-CSS avec tous les contacts et partenaires que tu veux, dans le bloc HTML de bas de page de la HOME où tu peux faire tout ce que tu veux.
